torch/jit/__init__.py

```diff
@@ -181,6 +182,27 @@ def _unique_state_dict(module, keep_vars=False):
     return filtered_dict
 
 
+def _create_interpreter_name_lookup_fn(frames_up=1):
+    def _get_interpreter_name_for_var(var):
```

Comment:
This function looks like that it could do some caching of `frame -> map[name, tensor]`.